modules/installation-registry-osp-creating-custom-pvc.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring_registry_storage/configuring-registry-storage.adoc
+
+[id="installation-registry-osp-creating-custom-pvc_{context}"]
+= Configuring an image registry with a custom back end on clusters that run on {rh-openstack}
```

Comment:
I would have gone with "Configuring an image registry with custom storage on clusters that run on {rh-openstack}".

"storage" seems to be the common term in the other topics in this section of the TOC.

Reply:
```suggestion
= Configuring an image registry with custom storage on clusters that run on {rh-openstack}
```